.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files

```

Comment:
@nelsonwong-jcc component looks good, but there are IDE folders ".idea" that were included in the commit.  Please remove them.

Reply:
Will do